### ISSUE TITLE
Installation of 2.54 fails on linux

### ISSUE THREAD

**Issue by lukehsiao** [netlify/cli#952]
---------------------------------------------------
BUG REPORT INFORMATION
---------------------------------------------------
**- Do you want to request a _feature_ or report a _bug_?**
Bug.

**- What is the current behavior?**
I'm unable to install the latest netlify-cli via npm.

**- If the current behavior is a bug, please provide the steps to reproduce.**

```
$ npm install netlify-cli -g
npm ERR! code ETARGET
npm ERR! notarget No matching version found for @netlify/cache-utils@^1.0.0.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.
npm ERR! notarget
npm ERR! notarget It was specified as a dependency of '@netlify/build'
npm ERR! notarget

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/pi/.npm/_logs/2020-06-24T16_16_06_507Z-debug.log
```

**- What is the expected behavior?**
The latest version of netlify-cli should be installed

**- Local Environment Information**
```
────────────────────┐
 Environment Info   │
────────────────────┘

  System:
    OS: Linux 4.19 Raspbian GNU/Linux 10 (buster) 10 (buster)
    CPU: (4) arm ARMv7 Processor rev 4 (v7l)
  Binaries:
    Node: 14.4.0 - /usr/bin/node
    npm: 6.14.5 - /usr/bin/npm
  Browsers:
    Firefox: 68.9.0esr
  npmGlobalPackages:
    netlify-cli: 2.50.0

netlify-cli/2.50.0 linux-arm node-v14.4.0
```

**Comment by ehmicky** [comment]
Hi @lukehsiao,

Can you try again?
According to `npm`, `@netlify/cache-utils@1.0.0` does exist: https://www.npmjs.com/package/@netlify/cache-utils

**Comment by lukehsiao** [comment]
I see, looks like it was just released ~30mins ago, even those v2.54 was release 9 days ago? Is there a way for netlify to make sure things are release in sync, so that the cli isn't pointing to non-existant dependencies for several days?

I can confirm that it installs correctly now that v1.0.0 of cache-utils exists.

**Comment by ehmicky** [comment]
> I see, looks like it was just released ~30mins ago, even those v2.54 was release 9 days ago? Is there a way for netlify to make sure things are release in sync, so that the cli isn't pointing to non-existant dependencies for several days?

This problem was not due to the release being 9 days ago, but a few-seconds-long lag from npmjs.com when running `npm publish`. We are looking into how to prevent this in the future.
